src/main/java/nextstep/subway/line/ui/LineController.java

```diff
@@ -50,6 +51,18 @@ public ResponseEntity deleteLine(@PathVariable Long id) {
         return ResponseEntity.noContent().build();
     }
 
+    @PostMapping(value = "{id}/sections")
+    public ResponseEntity addSectionToLine(@PathVariable Long id, @RequestBody SectionRequest sectionRequest) {
```

Comment:
```suggestion
    public ResponseEntity<Void> addSectionToLine(@PathVariable Long id, @RequestBody SectionRequest sectionRequest) {
```

이렇게 타입을 명시적으로 선언하면 더 좋을 것 같습니다 :)